src/ngraph/op/convolution.cpp

```diff
@@ -207,12 +207,12 @@ op::v1::ConvolutionBackpropData::ConvolutionBackpropData(const Output<Node>& dat
 const PartialShape op::v1::ConvolutionBackpropData::get_output_shape() const
 {
     PartialShape shape{vector<Dimension>(m_strides.size() + 2)};
-    auto data_pshape = get_input_partial_shape(0);
+    auto data_pshape = input(0).get_partial_shape();
```

Comment:
I decided `get_input_partial_shape(i)` is too convenient and have a PR that un-deprecates them, so its up to you if you want these changes or not.